src/test/java/seedu/guest/model/guest/PhoneTest.java

```diff
@@ -37,4 +39,11 @@ public void isValidPhone() {
         assertTrue(Phone.isValidPhone("93121534"));
         assertTrue(Phone.isValidPhone("124293842033123")); // long phone numbers
     }
+
+    @Test
+    public void hashcode() {
```

Comment:
Similarly, should we standardise the format for these tests with line spacing and comments?

Reply:
Agreed. Added line spacing and comments.